torch/nn/modules/module.py

```diff
@@ -86,6 +86,84 @@ def __setstate__(self, state: Dict):
             self.module = weakref.ref(state["module"])
 
 
+# N.B.: This calss is NOT deriving from `_WrappedHook`, because pre- and post-
```

Comment:
Yeah I had this same realisation. It does feel like they should be mergable, but I couldn't see a simple way to do so.